pages/_app.tsx

```diff
@@ -20,11 +20,7 @@ const App = ({ Component, pageProps, session }: ExpandedAppProps) => {
 				<div className="app">
 					<Header />
 					<div id="main-content" tabIndex={-1}>
-						{pageProps && pageProps.notFound ? (
-							<h1>404 Not Found</h1>
-						) : (
-							<Component {...pageProps} />
-						)}
```

Comment:
Even if for nothing else, getting to clean up this section is makes v10 worth it.